--- pykolab-0.6.5.orig/share/templates/imapd.conf.tpl	2013-08-22 15:26:51.000000000 +0100
+++ pykolab-0.6.5/share/templates/imapd.conf.tpl	2013-09-15 14:54:18.834592778 +0100
@@ -6,9 +6,9 @@
 sasl_pwcheck_method: auxprop saslauthd
 sasl_mech_list: PLAIN LOGIN
 allowplaintext: no
-tls_cert_file: /etc/pki/cyrus-imapd/cyrus-imapd.pem
-tls_key_file: /etc/pki/cyrus-imapd/cyrus-imapd.pem
-tls_ca_file: /etc/pki/cyrus-imapd/cyrus-imapd.pem
+tls_cert_file: /etc/ssl/private/cyrus-imapd.pem
+tls_key_file: /etc/ssl/private/cyrus-imapd.pem
+tls_ca_file: /etc/ssl/private/cyrus-imapd.pem
 # uncomment this if you're operating in a DSCP environment (RFC-4594)
 # qosmarking: af13
 auth_mech: pts
