--- pykolab-0.6.5.orig/pykolab/setup/setup_mta.py	2013-08-22 15:26:51.000000000 +0100
+++ pykolab-0.6.5/pykolab/setup/setup_mta.py	2013-09-15 14:55:48.455545959 +0100
@@ -395,6 +395,14 @@
             myaugeas.save()
         myaugeas.close()
 
+    if os.path.isfile('/etc/default/wallace'):
+        myaugeas = Augeas()
+        setting = os.path.join('/files/etc/default/wallace','START')
+        if not myaugeas.get(setting) == 'yes':
+            myaugeas.set(setting,'yes')
+            myaugeas.save()
+        myaugeas.close()
+
     if os.path.isfile('/bin/systemctl'):
         subprocess.call(['systemctl', 'restart', 'postfix.service'])
         subprocess.call(['systemctl', 'restart', 'amavisd.service'])
